tests/e2e/e2e_test.go

```diff
@@ -402,6 +402,12 @@ var _ = Describe("odoe2e", func() {
 				// remove the .odoignore file
 				Expect(os.Remove(ignoreFilePath)).To(BeNil())
 			})
+			// should be tested after the perviour It
+			It("should be able to push the local files on second iteration successfully", func() {
+				runCmdShouldPass("odo create nodejs:8 nodejs-doublepush --local " + tmpDir + "/nodejs-ex")
+				runCmdShouldPass("odo push")
+				runCmdShouldPass("odo push")
```

Comment:
Wouldn't it be good to delete the component at the end of this test?

Reply:
yes sure i will add that

Reply:
its been added here https://github.com/redhat-developer/odo/pull/1448/commits/16cdcefeea023b0e83560041ab47c7eaf0b74516

Reply:
Can you capture and validate the test result for the the second push.